src/components/DiseaseTable.tsx

```diff
@@ -103,6 +105,29 @@ const DiseaseTable = ({
         ),
     };
 
+    const handleCellInHoveredRow = (
+        record: UnpackedDiseaseCellLine,
+        index: number | undefined
+    ) => {
+        const hoveredAndDataComplete =
+            index === hoveredRowIndex && record.status === "data complete";
+        return {
+            className: hoveredAndDataComplete ? hoveredRow : "",
+            onMouseEnter: () => {
+                if (index !== undefined) {
+                    setHoveredRowIndex(index);
+                }
+            },
+            onMouseLeave: () => {
+                setHoveredRowIndex(-1);
+            },
+            onClick: () => {
+                if (hoveredAndDataComplete)
```

Comment:
this can just be `if (record.status === CellLineStatus.DataComplete` because if you're clicking it, it's also hovered. I also wonder if this would be broken on tablets since there really isn't a hover state. It's safer to have the if statement be only for what you truly need to be true